keyboards/uno/config.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2020 Snipeye
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0xACC8
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Broekhuijsen
```

Comment:
Shouldn't this be:

```suggestion
#define MANUFACTURER    Snipeye
```

?

Reply:
I can change it; but it's true either way - I go by both online, so...

Reply:
Oh, I wasn't aware that you *were* @Snipeye – you've committed here from that account before. Feel free to disregard this if you wish.

Reply:
I am he.  He is me?  We are all together.  If I plan on disregarding that, is this ready for merge?